index.test-d.ts

```diff
@@ -1,13 +1,22 @@
+import {Buffer} from 'node:buffer';
+// For some reason a default import of `process` causes
+// `process.stdin`, `process.stderr`, and `process.stdout`
+// to get treated as `any` by `@typescript-eslint/no-unsafe-assignment`.
+import * as process from 'node:process';
```

Comment:
Not sure how to best debug this, but I couldn't use

```js
import process from 'node:process';
```

as documented at https://nodejs.org/api/process.html#process without running into TypeScript issues.

Reply:
Thanks @NMinhNguyen. I think the code comment will do then :+1: 
Especially considering this is only a test file, and only for types.